packages/jsii-pacmak/lib/packaging.ts

```diff
@@ -52,15 +55,20 @@ export class JsiiModule {
   /**
    * Prepare an NPM package from this source module
    */
-  public async npmPack() {
+  public async npmPack(packCommand = DEFAULT_PACK_COMMAND) {
     this._tarball = await Scratch.make(async (tmpdir) => {
-      // Quoting (JSON-stringifying) the module directory in order to avoid
-      // problems if there are spaces or other special characters in the path.
-      const args = ['pack', JSON.stringify(this.moduleDirectory)];
+      const args = packCommand.split(' ');
```

Comment:
This is dangerous... if a customer uses a fully-qualified path that includes white space, this will always fail...

Reply:
updated in latest rev.